armi/reactor/geometry.py

```diff
@@ -519,9 +521,8 @@ def checkValidGeomSymmetryCombo(
 REFLECTIVE = "reflective"
 PERIODIC = "periodic"
 NO_SYMMETRY = "no symmetry"
-THROUGH_CENTER_ASSEMBLY = (
-    "through center assembly"  # through center assembly applies only to cartesian
-)
+# through center assembly applies only to cartesian
+THROUGH_CENTER_ASSEMBLY = "through center"
```

Comment:
Should we rename to just `THROUGH_CENTER`? It seems that this symmetry could be applied to block grids as well, no?

Reply:
Yeah, I guess so. Since we no longer store this stuff in strings/do direct string comparisons against them, `THROUGH_CENTER_ASSEMBLY` is not really used anywhere other than in this module. Might as well rename though

Reply:
Ugg it is defined in geometry.py, but still being used directly in one place in blueprints. Blueprints shouldn't be using it directly, but im not sure this is the time or place

Reply:
Agreed that its not the right time/place, but just something i noticed...